.isort.cfg

```diff
@@ -10,4 +10,3 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
-known_third_party=babel,dateutil,openupgradelib
```

Comment:
Copier deleted this line according to the template, but I am not sure if this should be added back.

Reply:
It's not needed.